ios/Classes/SwiftVibrationPlugin.swift

```diff
@@ -1,29 +1,171 @@
 import Flutter
 import UIKit
 import AudioToolbox
+import CoreHaptics
 
+@available(iOS 13.0, *)
```

Comment:
Could annotation the entire class break something?

Reply:
I honestly don’t know.